azure-kusto-data/source/security.js

```diff
@@ -71,7 +71,19 @@ module.exports = class AadHelper {
         }
     }
 
-    getAuthHeader(cb) {
+    _getAuthHeader() {
```

Comment:
What is this? Aren't you missing a function definition? Can it also be marked as async?

Reply:
"What is this? Aren't you missing a function definition?" - ?
" Can it also be marked as async?" - probably yes, but not have to AFAIK